tool/e2e/.github/workflows/main.yaml

```diff
@@ -0,0 +1,18 @@
+name: ci
```

Comment:
What ci strategy do we want for e2e tests?

Reply:
wait, shouldnt `.github` be in the root of the repo?

Reply:
Also, naming here could be "e2e" 😂

Reply:
Yes it should, and yea lets just have a `e2e.yaml` with a `e2e` name so it shows up nicely in the UI on github

Reply:
Yes it should be at root, this is just the file that was generated when creating the dart package.